contracts/src/LightClient.sol

```diff
@@ -64,17 +68,33 @@ contract LightClient {
     error MissingLastBlockForCurrentEpoch(uint64 expectedBlockHeight);
     /// @notice Invalid user inputs: wrong format or non-sensible arguments
     error InvalidArgs();
+    /// @notice Wrong plonk proof or public inputs.
+    error InvalidProof();
 
     constructor(LightClientState memory genesis, uint32 numBlockPerEpoch) {
-        if (genesis.viewNum != 0 || genesis.blockHeight != 0) {
+        // stake table commitments and threshold cannot be zero, othrewise it's impossible to
```

Comment:
```suggestion
        // stake table commitments and threshold cannot be zero, otherwise it's impossible to
```

Reply:
done in [6171539](https://github.com/EspressoSystems/espresso-sequencer/pull/854/commits/61715390c68c49dcf05f32a2505fc782957d9628)